services/app-api/utils/kafka/kafka-source-lib.ts

```diff
@@ -0,0 +1,200 @@
+/* eslint-disable no-console */
+import AWS from "aws-sdk";
+import s3Lib from "../s3/s3-lib";
+import { Kafka } from "kafkajs";
+import { S3EventRecord } from "aws-lambda";
+
+const STAGE = process.env.STAGE;
+const brokerStrings = process.env.BOOTSTRAP_BROKER_STRING_TLS
+  ? process.env.BOOTSTRAP_BROKER_STRING_TLS
+  : "";
+
```

Comment:
Should this library check to see if the environment variables are missing and throw an error if they are?

Reply:
Updated to error